release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2077,4 +2077,31 @@ A new conditional gatherer has been implemented for the `SamplesImagestreamImpor
 [id="ocp-4-9-4-upgrading"]
 ==== Upgrading
 
-To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-upgrading-cli_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
+[id="ocp-4-9-5"]
+
+=== RHBA-2021:4005 - {product-title} 4.9.5 bug fix update
+
+Issued: 2021-11-01
+
+{product-title} release 4.9.5 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:4005[RHBA-2021:4005] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2021:4004[RHSA-2021:4004] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6465271[{product-title} 4.9.4 container image list]
+
+[id="ocp-4-9-5-known-issues"]
+==== Known Issues
+
+* The flag that is used to hide or show the *Storage -> Overview* page in the {product-title} web console is misconfigured. Consequently, the *Overview* page is invisible after deploying a cluster that included OpenShift Cluster Storage. A fix for this bug is planned for a future release. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2013132[*BZ#2013132*])
+
+[id="ocp-4-9-5-bug-fixes"]
+==== Bug fixes
+
+* With the deprecation of the last triggered image ID field for build configs, the image change trigger controller stopped checking the ID field prior to initiating builds. Consequently, if a build config was created and had an image change trigger start while the cluster was a level prior to {product-title} 4.8, it continuously tried to trigger builds. With this update, unnecessary attempts to trigger builds no longer occur. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2006791[*BZ#2006791*])
+
+
+[id="ocp-4-9-5-upgrading"]
+==== Upgrading
+
+To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster by using the CLI] for instructions.
```

Comment:
Same comment as previously mentioned. I think we're targeting 4.9 z-stream -> 4.9 z-stream upgrades here.